lib/decorators-callbacks.js

```diff
@@ -18,11 +18,24 @@ module.exports.decorateLambdaReqCallback = function(getPrincipalId) {
   };
 };
 
-module.exports.decorateAddCORSCallback = function() {
+module.exports.decorateAddCORSCallback = function(corsOpts, isOptionMethod) {
+  // TODO: get default cors options from variables
+  const cors = {
+    origins: ['*'],
+    methods: ['GET', 'PUT', 'HEAD', 'PATCH', 'POST', 'DELETE', 'OPTIONS'],
+    headers: ['Authorization', 'Content-Type', 'x-amz-date', 'x-amz-security-token']
+  };
+  if(corsOpts instanceof Object) {
+    Object.assign(cors, corsOpts);
+  } else if(corsOpts != true) {
+    throw new Error('Invalid cors configuration!');
+  }
   return function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', '*');
-    res.header('Access-Control-Allow-Methods', 'GET,PUT,HEAD,PATCH,POST,DELETE,OPTIONS');
-    res.header('Access-Control-Allow-Headers', 'Authorization,Content-Type,x-amz-date,x-amz-security-token');
+    if(isOptionMethod) {
```

Comment:
Why should we not have this? I'd rather use the corsOpts to select which header to set